PinwheelSDK/Classes/Pinwheel.swift

```diff
@@ -99,9 +133,37 @@ public class PinwheelViewController: UIViewController, WKUIDelegate, WKScriptMes
         }
         return nil
     }
+    
+    private func bodyDataFromMessage(_ message: WKScriptMessage) -> Data? {
+        if let bodyString = message.body as? String,
+           let bodyData = bodyString.data(using: .utf8) {
+
+            return bodyData
+        }
+        return nil
+    }
+}
+
+private enum PinwheelEventHandler: String {
+    case openEventHandler
```

Comment:
Same here. We should stick to loginEventHandler, exitEventHandler, successEventHandler, errorEventHandler, and genericEventHandler.